src/settings/VideoSettings.tsx

```diff
@@ -46,30 +57,70 @@ export default function VideoSettings() {
       noValidate
       autoComplete="off"
     >
-      <Box component="span" sx={{ display: 'flex', alignItems: 'center'}}>
-        <FormControl sx={{my: 1}}>
-          <InputLabel id="demo-simple-select-label" sx = {style}>Monitor</InputLabel>
-          <Select
-            labelId="demo-simple-select-label"
-            id="monitor-index"
-            value={config.monitorIndex}
-            label="Monitor"
-            onChange={(event) => modifyConfig('monitorIndex', event.target.value)} 
-            sx={style}
-          >
-            { displayConfiguration.map((display: OurDisplayType) =>
-              <MenuItem key={ 'display-' + display.id } value={ display.index + 1 }>
-                [{ display.index + 1 }] { display.size.width }x{ display.size.height } @ { display.displayFrequency } Hz ({display.physicalPosition}) {display.primary ? ' (Primary)' : ''}
-              </MenuItem>
-            )}
-          </Select>
-        </FormControl>
-        <Tooltip title={configSchema["monitorIndex"].description} >
-          <IconButton>
-            <InfoIcon style={{ color: 'white' }}/>
-          </IconButton>
-        </Tooltip>
-      </Box>
+      <FormControl fullWidth>
```

Comment:
Seem to have lost the MUI box element here. I think that was crucial for aligning things when I made the settings changes recently. Is that deliberate? Does it still look OK? Also lost some of the margin specification. Suspect a conflict resolution gone the wrong direction?

Reply:
That's not inconceivable. I'll have a look at it.